src/reynir/bintokenizer.py

```diff
@@ -412,6 +415,7 @@ def all_except_suffix(s: str) -> str:
     ("in", "a", "an", "for", "and", "the", "for", "on", "of")
 )
 
+SENTENCE_END_START_TOKENS: FrozenSet[int] = frozenset((TOK.S_END, TOK.S_BEGIN))
```

Comment:
Nitpick: A better name would be SENTENCE_DELIMITER_TOKENS

Reply:
Fixed!